src/main/resources/ath-container/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN apt-get clean && \
         git \
         imagemagick \
         iptables \
-        firefox-esr \
+        firefox \
```

Comment:
https://seleniumhq.wordpress.com/2017/08/09/firefox-55-and-selenium-ide/

xenial seems to use [56+](https://launchpad.net/ubuntu/xenial/+source/firefox) so using esr is a good idea (if it exists...)